packages/whats-new/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+	rules: {
+		'react/react-in-jsx-scope': 0,
+	},
```

Comment:
This shouldn't be necessary, is it?

Reply:
nope, there were a couple changes that I hadn't saved, about to update them